examples/rl/deep_q_network_breakout.py

```diff
@@ -209,13 +214,13 @@
                 q_action = tf.reduce_sum(tf.multiply(q_values, masks), axis=1)
                 # Calculate loss between new Q-value and old Q-value
                 # Clip the deltas using huber loss for stability
-                loss = tf.reduce_mean(keras.losses.huber(updated_q_values, q_action))
+                loss = tf.reduce_mean(keras.losses.Huber()(updated_q_values, q_action))
```

Comment:
Maybe I am wrong but I don't think you need to initiate the class?
See here:
https://keras.io/api/losses/regression_losses/#huber-function

Reply:
It should be fine in general, but at least in the release version of TF2.2, the tf.keras.losses.huber function seems to be not exposed [See here](https://github.com/tensorflow/tensorflow/blob/r2.2/tensorflow/python/keras/losses.py).

In the main branch of TF they seem to have added it though [See here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/losses.py#L1421)

Reply:
Cool, they will both work :D thanks!

Reply:
If you want to use `Huber()`, then:

1. it should be instantiated outside of the loop, and then you should reuse the same instance
2. the `reduce_mean` is no longer necessary